vsts/node-nightly-windows.yaml

```diff
@@ -132,12 +134,21 @@ stages:
         pathToPublish: $(Build.ArtifactStagingDirectory)
         artifactName: azure-iot-dps-node
 
-  # - task: AzureFileCopy@2
-  #   displayName: 'Copy Release Artifacts to Partner Release Pipeline'
-  #   condition: ${{ parameters.release }}
-  #   inputs:
-  #     SourcePath: $(Build.ArtifactStagingDirectory)\_out
-  #     azureSubscription: azuresdkpartnerdrops
-  #     Destination: AzureBlob
-  #     storage: azuresdkpartnerdrops
-  #     ContainerName: 'drops/azure-iot-sdk/node/'
+    # copy packed npm archive dated folder for release archive
+    - task: CopyFiles@2
+      displayName: 'Copy packed archive to artifact staging'
+      inputs:
+        contents: $(package)
+        targetFolder: $(Build.ArtifactStagingDirectory)\out\$(buildFolder)
+        sourceFolder: $(Build.ArtifactStagingDirectory)
+
+    # copy to SDK partner release storage
```

Comment:
Typically the sbom has to be inspected prior to publishing/distributing for other user/system usage, does this pipeline take that into account? Or are there further steps involved before "final" publish?

Reply:
This pipeline copies the packed NPM archive into a partner SDK storage bucket - but nothing is released automatically.

To release the package to NPM, there's a separate build pipeline that takes a path to this specific folder (`drops/azure-iot-sdk/node/{date}`) that actually runs the `npm publish` step.

We could either:
 - Add an approval step in between the push to the storage bucket (for SBOM review) to make it more like our other pipelines
 - Check SBOM before running the actual "release / publish" pipeline
 
I'm actually in favor of the former (approval step in this pipeline) given that once it's "ready to release" we should know that the SBOM is checked already - so any artifact that makes it's way to the "release staging" storage is ready to ship.

Reply:
Alright, yea I'm good with the first option as well.